cinema/serializers.py

```diff
@@ -125,9 +125,9 @@ class TicketSerializer(serializers.ModelSerializer):
     def validate(self, attrs):
         data = super(TicketSerializer, self).validate(attrs=attrs)
         Ticket.validate_ticket(
-            attrs["row"], 
-            attrs["seat"], 
-            attrs["movie_session"].cinema_hall, 
+            attrs["row"],
+            attrs["seat"],
+            attrs["movie_session"].cinema_hall,
             ValidationError
         )
```

Comment:
In the `validate` method of `TicketSerializer`, ensure that `row`, `seat`, and `movie_session` are present in `attrs` before accessing them to avoid potential `KeyError`. Consider using `attrs.get('row')` and similar for other fields with appropriate default values or error handling.